Sources/TuistGenerator/Generator/ConfigGenerator.swift

```diff
@@ -303,6 +307,31 @@ final class ConfigGenerator: ConfigGenerating {
         return settings
     }
 
+    private func swiftMacrosDerivedSettings(
+        target: Target,
+        graphTraverser: GraphTraversing,
+        projectPath: AbsolutePath
+    ) -> SettingsDictionary {
+        let targets = graphTraverser.directSwiftMacroFrameworkTargets(path: projectPath, name: target.name)
+        if targets.isEmpty { return [:] }
+        var settings: SettingsDictionary = [:]
+        settings["OTHER_SWIFT_FLAGS"] = .array(targets.flatMap { target in
+            let macroExecutables = graphTraverser.directSwiftMacroExecutables(path: target.path, name: target.target.name)
+            return macroExecutables.flatMap { macroExecutable in
+                switch macroExecutable {
+                case let .product(_, productName, _):
+                    return [
+                        "-load-plugin-executable",
+                        "$BUILT_PRODUCTS_DIR/\(target.target.productNameWithExtension)/Macros/\(productName)/#\(productName)",
```

Comment:
ref: https://github.com/apple/swift/blob/main/include/swift/Option/Options.td#L1944C26-L1944C26

Reply:
🙏🏼 I didn't know about the existence of that. Thanks for sharing @muukii